src/soilbiogeochem/SoilBiogeochemNitrifDenitrifMod.F90

```diff
@@ -413,9 +421,24 @@ subroutine SoilBiogeochemNitrifDenitrif(bounds, num_bgc_soilc, filter_bgc_soilc,
             wfps_vr(c,j) = max(min(h2osoi_vol(c,j)/watsat(c, j), 1._r8), 0._r8) * 100._r8
             fr_WFPS(c,j) = max(0.1_r8, 0.015_r8 * wfps_vr(c,j) - 0.32_r8)
 
-            ! final ratio expression 
-            n2_n2o_ratio_denit_vr(c,j) = max(0.16_r8*ratio_k1(c,j), ratio_k1(c,j)*exp(-0.8_r8 * ratio_no3_co2(c,j))) * fr_WFPS(c,j)
+            if (use_soil_nox) then
+            ! final ratio expression  
+            !DAYCENT4.5 EQUATION modified n2_n2o_ratio_denit to be consistent with daycent code
+               fno3_co2 = max(0.16_r8*ratio_k1(c,j), ratio_k1(c,j)*exp(-0.8_r8 * ratio_no3_co2(c,j)))
+               n2_n2o_ratio_denit_vr(c,j) = max(0.1_r8,fno3_co2*fr_WFPS(c,j))
 
+               afps_vr(c,j) = 1._r8-max(min(h2osoi_vol(c,j)/watsat(c,j), 1._r8), 0._r8)
+               Dr = 0.209_r8*afps_vr(c,j)**(4._r8/3._r8)
+               
+               nox_n2o_ratio_vr(c,j) = 15.2_r8 + (35.5_r8 * atan(0.68_r8 * rpi * (10.0_r8 * Dr - 1.86_r8))) / rpi
```

Comment:
This is another area with numerous hardwired values that need some kind of documentation, such as assigning variable names and literature refs to them.